security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -629,9 +638,16 @@ func (sc *SecretCache) rotate(updateRootFlag bool) {
 		// Re-generate secret if it's expired.
 		if sc.shouldRotate(&secret) {
 			atomic.AddUint64(&sc.secretChangedCount, 1)
-			// Send the notification to close the stream if token is expired, so that client could re-connect with a new token.
-			if sc.isTokenExpired(&secret) {
-				cacheLog.Debugf("%s token expired", logPrefix)
+			if SupportRotationWithCert {
```

Comment:
it's probably more proper to change the variable (and the env) to `UseRorationWithCert`, the `Support` is more about the capability instead of the behavior.

Reply:
yes will refactor that